### DISCUSSION TITLE
Tbl_site_references biblio quality assurance

### DISCUSSION THREAD

**Discussion by Loer0022** [humlab-sead/sead_change_control#303] [General]
Further quality assurance and corrections is needed in tbl_site_references to make sure that the right site is connected to its respective biblio_id but also to make sure that there are posts to accommodate for such connections to be made. This would then make the SEAD bibliography more complete and allow for the database to provide the end users with both reports about the data but also excavations or studies made related to the data on the site.

This issue is more of a reminder of what work is still needed for the tbl_biblio quality control to 
get a general overview of the current situation and making publications avaliable to the end users.

**Comment by roger-mahler** [comment]
Closing this issue.
